### ISSUE TITLE
[Designer]Bug 35714425: [V1.0-> Scenarios - FoodOrder.json]: "Your registration is almost complete" is not defined as a heading on 'FoodOrder.Json' card.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7620]
### Problem Description

[Bug 35714425](https://microsoft.visualstudio.com/OS/_workitems/edit/35714425): [V1.0-> Scenarios - FoodOrder.json]: "Your registration is almost complete" is not defined as a heading on 'FoodOrder.Json' card.

Test Environment:
Device: iPhone x
iOS: 14.7.1
Application: Adaptive Cards
App Version: Version 1.0 (2.3.1-beta.20210823.1)
Screen Reader: Voice Over

Repro Steps:
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.0" control and select it.
4. Navigate on  "V1.0" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "FoodOrder.json" control and select it.
7. Navigate the 'FoodOrder.json' screen and enable Voiceover for the ease of access.
8. Navigate till 'Your registration is almost complete' and hear screen reader narration.

Actual Result:
"Your registration is almost complete" is not defined as a heading on 'FoodOrder.Json' card. Screen reader is only announcing as, "Your registration is almost complete text field double tap to edit Use the rotor to access Misspelt Words".

Expected Result:
"Your registration is almost complete" should be defined under a heading. Screen reader should announce as, "Your registration is almost complete heading Use the rotor to access headings".

Note: 
Same issue is also repro for 'Malt and Vine' text on FoodOrder.json card. (Please refer attachment)
v1.0 -Element -AdaptiveCard.json- Issue is repro for 'Publish Adaptive Card Schema'. (Please refer attachment)
v1.0-Element-ColumnSet.Spacing.json- issue is repro for 'ColumnSet Spacing'. (Please refer attachment)
v1.0-Element-Container.Spacing.json- issue is repro for 'Container Spacing'.
Issue is repro throughout application. Headings is not defined as heading.
User Experience:
It will be confusing & difficult for the users if Heading will not be defined inside the adaptive card.

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Card JSON

```JSON
Also, repro on UWP
```

**Comment by vagpt** [comment]
This issue is not repro on the environment below, hence closing this issue.

**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.23.10.15.1)
Screen Reader: Voice Over


https://github.com/microsoft/AdaptiveCards/assets/95913748/a7d669dd-c48f-4b7c-bdec-3706d99a0c1a
